packages/next/trace/trace.ts

```diff
@@ -1,10 +1,9 @@
-import { randomBytes } from 'crypto'
-import { SpanId } from './shared'
+import { customAlphabet } from 'next/dist/compiled/nanoid/index.cjs'
 import { reporter } from './report'
 
 const NUM_OF_MICROSEC_IN_SEC = BigInt('1000')
-
-const getId = () => randomBytes(8).toString('hex')
+const nanoid = customAlphabet('0123456789abcdefghijklmnopqrstuvwxyz', 8)
+const getId = () => Buffer.from(nanoid(), 'utf8').toString('hex')
```

Comment:
```suggestion
const nanoid = customAlphabet('0123456789abcdef', 16)
const getId = () => nanoid()
```

Reply:
Using nanoid does not work because it includes - and _ as well as capitals which is not supported by jaeger.